rules/use-t.js

```diff
@@ -12,7 +12,7 @@ module.exports = function (context) {
 
 			var functionArg = node.arguments[node.arguments.length - 1];
 
-			if (!functionArg.params || !functionArg.params.length) {
+			if (!functionArg || !functionArg.params || !functionArg.params.length) {
```

Comment:
With so many negated arguments, I think this would be clearer:

```js
if (!(functionArg && functionArg.params && functionArg.params.length)) {
```